superset-frontend/packages/superset-ui-chart-controls/test/components/labelUtils.test.tsx

```diff
@@ -18,6 +18,9 @@
  */
 import React from 'react';
 
+// @ts-ignore
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { render, screen } from 'spec/helpers/testing-library';
```

Comment:
Should we bring `RTL` into Superset-ui ?

Reply:
I think it makes more sense to test this component with RTL rather than deeply comparing JS objects created by React. However, like Michael noticed in [a comment in different PR](https://github.com/apache/superset/pull/19236/files/f5d4d7c05f43a69cdc4fa584e0163a7a56c10c99#r830264183), we shouldn't import from `spec/` in `superset-ui`. I'll import RTL directly until we solve the problem of cyclic dependencies